mne/viz/topomap.py

```diff
@@ -331,6 +331,7 @@ def plot_projs_topomap(projs, layout=None, cmap=None, sensors=True,
                     shape = (len(idx) // 2, 2, -1)
                     pos = pos.reshape(shape).mean(axis=1)
                     data = _merge_grad_data(data[grad_pairs]).ravel()
+                    assert (data > 0).all()  # Either this breaks, or the plot should be all red !!
```

Comment:
It makes no sense that this works and the gradients look like this:

![image](https://user-images.githubusercontent.com/7044835/47349903-e76df300-d6b4-11e8-8079-e769e2dd0078.png)